.github/workflows/alas-issues.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: "[ALAS] Open issues for new bulletins"
+on:
+  workflow_dispatch:
+    inputs:
+      window:
+        description: "Only consider bulletins published within this relative time window (golang Duration)"
+        default: "24h"
+        required: true  
+  schedule:
+    # once an hour, at the top of hour
+    - cron: "0 0 * * *"
+permissions:
+  issues: write
+jobs:
+  alas-al2-bulletins:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: guilhem/rss-issues-action@0.5.2
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          feed: "https://alas.aws.amazon.com/AL2/alas.rss"
+          dry-run: "false"
+          lastTime: "${{ github.event.inputs.window || '24h' }}"
+          labels: "alas,alas/al2"
+          titleFilter: "(medium|low)"
```

Comment:
I'd expect to see the string "Important" appear somewhere in this file. Why does it not?

Reply:
If the `titleFilter` matches, the feed item is *excluded* (it's a negative filter).

One improvement we could PR to this existing Action (or make in our own) would be positive filters (see my comment about only including bulletins for certain packages, though I could argue either way on that).

Reply:
Ah so "Important" includes "Critical". got it.